package.json

```diff
@@ -20,7 +20,6 @@
     "jsoneditor": "^9.1.9",
     "jsoneditor-react": "^3.1.0",
```

Comment:
hi @rajatkhade 

Connect with @nihal772 

to make same package.json with him I have reviewed his package.json